### ISSUE TITLE
Sound mute button doesn't activate the LED on ThinkPad E565 - Lenovo

### ISSUE THREAD

**Issue by pbrilius** [pop-os/beta#328]
**How did you upgrade to 21.10? (Fresh install / Upgrade)**

 
Fresh one.
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

gnome-shell:
  Įdiegta:    40.5-1ubuntu2pop0~1633630918~21.10~91d3049
  Kandidatas: 40.5-1ubuntu2pop0~1633630918~21.10~91d3049
  Versijų lentelė:
 *** 40.5-1ubuntu2pop0~1633630918~21.10~91d3049 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
        100 /var/lib/dpkg/status
     40.5-1ubuntu2 500
        500 http://us.archive.ubuntu.com/ubuntu impish/main amd64 Packages

**Issue/Bug Description:**

Turning off the sound - _muting_ it - by the built in Lenovo _E565_ SMB ThinkPad button doesn't allow the **LED** to indicate its status - sound is muted, of course, though the **LED** is irresponsibly non-responsive to the sound-muted status-on disposition.

**Steps to reproduce (if you know):**

Launch _POP_! beta live **ISO** on Lenovo **E565** ThinkPad laptop and try sound off by clicking the _corresponding_ Fn+F1 button, where it stands for it.

**Expected behavior:**

I'd think that the **LED** should be turned on _leaving_ the status indicating sound off when **clicked** from the _status_ on and vice-versa.

**Other Notes:**

**Comment by jacobgkau** [comment]
Does this issue also occur on 21.04?

**Comment by pbrilius** [comment]
No, it's OK with the _previous_ versions like 21.04/

**Comment by pbrilius** [comment]
Yes, I performed an installation, I use it daily and the _corresponding_ button isn't working _correctly_. I have fresh installed it, though I'm **emphasizing** the easy way to reproduce the bug on the actual hardware w\o\ a virtualization option.

**Comment by pbrilius** [comment]
Actually, after a while, using this _OS_ and upgrading regularly, it occurred again - the **LED** is not reactive.

**Comment by pbrilius** [comment]
Seems legit that now it works.

**Comment by pbrilius** [comment]
I figured it out - that only on Bluetooth connected devices this issue occurs, can you fix it please? I mean that provided a Bluetooth speaker is connected, the LED is not-responding.
